html/manifest/icons.json

```diff
@@ -19,15 +19,7 @@
               "version_added": null
             },
             "firefox_android": {
-              "version_added": "53",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "manifest.install.enabled",
-                  "value_to_set": "true"
-                }
-              ],
-              "notes": "By default, used for as the source for \"Add to home screen\" (see bug 1234558)."
+              "version_added": false
```

Comment:
Desktop is null above. Should it be?

Reply:
Desktop support is probably `false` as well, but I'm not totally confident on that.  I'd like to hold off digging into it until we commence our no-nulls project for HTML though?